docs/hooks-js-sandbox.md

```diff
@@ -0,0 +1,101 @@
+# JavaScript Hooks In Sandbox Mode
+
+## Usage
+
+```
+$ dredd apiary.apib http://localhost:3000 --sandbox --hookfiles=./hooks*.jss
+```
+
+### Dredd JS API Option
+
+Sndbox mode can be enabled in Dredd JavaScript API
+
+```javascript
+var Dredd = require('dredd');
+var configuration = {
+  server: "http://localhost",
+  options: {
+    path: "./test/fixtures/single-get.apib",
+    sandbox: true,
+    hookfiles: ['./test/fixtures/sandboxed-hook.js']
+  }
+};
+var dredd = new Dredd(configuration);
+
+dredd.run(function (error, stats) {
+  // your callback code here
+});
+```
+
+## Sandboxed JavaScript Hooks API reference
+
+The Sandbox mode can be used for running untrusted hook code. It can be activated with a CLI switch or with the JS API.
+In each hook file you can use following functions:
+
+`before(transactionName, function)`
+
+`beforeValidation(transactionName, function)`
+
+`after(transactionName, function)`
+
+`beforeAll(function)`
+
+`afterAll(function)`
+
+`beforeEach(function)`
+
+`beforeEachValidation(function)`
+
+`afterEach(function)`
+
+`log(string)`
+
+- A [Transaction Object](#transaction-object-structure) is passed as a first argument to the hook function for `before`, `after`, `beforeEach`, and `afterEach`.
+- An array of Transaction Objects is passed to `beforeEach` and `afterEach`.
```

Comment:
should `beforeEach` or `afterEach` be `beforeAll` or `afterAll` respectively?

`beforeEach` receives one transaction object